### PR TITLE
[Bug-7389][docs-zh][docs] Fix System User Manual link failure in version 2.0.1 and version 2.0.0

### PR DESCRIPTION
**Pull request by uestczhuhao**
Fix System User Manual link failure in version 2.0.1 and version 2.0.0. Close apache/dolphinscheduler#7389

I also fix incorrect Quick Start hyperlink in  /docs/en-us/2.0.0/user_doc/guide/system-manual.md, it mistake link to version 1.3.9

### CODE REVIEW

**Comment by zhongjiajie** [comment]
Hi @uestczhuhao, we do not forget to add `system-manual.md` file, we split the doc to multiply files. for example `system-manual.md` have doc to desc how to use our task type, but we add an independent part named `Task`
image

So in this situation, I think we should not revert doc `system-manual.md`.

File: docs/en-us/2.0.0/user_doc/architecture/design.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ scheduling system
 * **UI**
 
   The front-end page of the system provides various visual operation interfaces of the system,See more
-  at<a href="/en-us/docs/2.0.0/user_doc/system-manual.html" target="_self"> System User Manual </a>section。
+  at [System User Manual](../guide/system-manual.md) section。
```
Comment (zhongjiajie) [inline docs/en-us/2.0.0/user_doc/architecture/design.md:81]: And for this dead link, maybe we better to change it to `../guide/homepage.md` and the content change to `Introduction to Functions`
image

**Comment by zhongjiajie** [comment]
@uestczhuhao Are you still WIP?

**Comment by uestczhuhao** [comment]
> @uestczhuhao Are you still WIP?

yes, a little busy these days, i will fix and update pr tonight

**Comment by zhongjiajie** [comment]
We looking forward your contribution ❤️

File: docs/zh-cn/2.0.1/user_doc/architecture/design.md [hunk 2]
```diff
@@ -68,7 +68,7 @@
 
 * **UI** 
 
-    系统的前端页面，提供系统的各种可视化操作界面，详见<a href="/zh-cn/docs/2.0.1/user_doc/system-manual.html" target="_self">系统使用手册</a>部分。
+    系统的前端页面，提供系统的各种可视化操作界面，详见[功能介绍](../guide/homepage.md)部分。
```
Comment (zhongjiajie) [inline docs/zh-cn/2.0.1/user_doc/architecture/design.md:71]: I found it also in dev doc https://github.com/apache/dolphinscheduler-website/blob/d423ef4bf7b9a7a6f48cf5bab5d6a100d42122a1/docs/zh-cn/dev/user_doc/architecture/design.md#L102. could your also change `dev` zh and en doc? @uestczhuhao

**Review by zhongjiajie — COMMENTED** [review]
Seem you missing dev doc here

File: docs/zh-cn/2.0.1/user_doc/architecture/design.md (hunk 2 above)
Comment (uestczhuhao) [reply docs/zh-cn/2.0.1/user_doc/architecture/design.md:71]: done @zhongjiajie

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks for your contribution 👍

**Pull request merged**